nestoria-project/src/app/modules/property-listing-page/property-listing.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="nav">
+    <div>
+      <h2>Property Details</h2>
+    </div>
+    <div>
+      <app-button [id]="'button-plus'"
```

Comment:
Most common code style for Html structure with angular looks like - 
```
<div>
  <app-button 
    [id]="'button-plus'"
    label="Add"
    click="addToFavor()">   // <--- here use (click) instead of click, becose (click) trigger angular change detection
  </app-button>
</div>

```

Follow this parameters list  -
1) ([]) - bindings - like mode
2) () - actions line (click)='someMethod'
3) [] - simple values
4) default atributes - like type / class and etc

Reply:
fixed